SharpPcap/WinDivert/WinDivertDevice.cs

```diff
@@ -156,29 +182,33 @@ public ulong GetParam(WinDivertParam param)
             return value;
         }
 
-        public void SendPacket(RawCapture p)
-        {
-            if (p is WinDivertCapture packet)
-            {
-                WinDivertAddress address = default;
-                address.IfIdx = packet.InterfaceIndex;
-                address.SubIfIdx = packet.SubInterfaceIndex;
-                address.Flags = packet.Flags;
-                SendPacket(new ReadOnlySpan<byte>(p.Data), address);
-                return;
-            }
-            SendPacket(new ReadOnlySpan<byte>(p.Data));
-        }
-
-        public void SendPacket(ReadOnlySpan<byte> p)
+        public void SendPacket(RawCapture p, ICaptureHeader captureHeader)
```

Comment:
Is it just me or this method implementation is always the same for all devices?
If yes, would it be better to make it an extension method?